bbb-common-web/src/main/java/org/bigbluebutton/api/model/request/CreateMeeting.java

```diff
@@ -31,7 +31,6 @@
     @MeetingIDConstraint
     private String meetingID;
 
-    //@NotEmpty(message = "You must provide a voice bridge")
     @IsIntegralConstraint(message = "Voice bridge must be a 5-digit integral value")

```

Comment:
I don't know where this "Voice bridge must be a 5-digit integral value" message came from, but it's not correct; longer voice bridge strings are often used. Maybe state a minimum of 5 digits?